services/graph/pkg/service/v0/service.go

```diff
@@ -46,6 +47,21 @@ type Service interface {
 	PostGroupMember(http.ResponseWriter, *http.Request)
 	DeleteGroupMember(http.ResponseWriter, *http.Request)
 
+	GetSchools(http.ResponseWriter, *http.Request)
+	GetSchool(http.ResponseWriter, *http.Request)
+	PostSchool(http.ResponseWriter, *http.Request)
+	PatchSchool(http.ResponseWriter, *http.Request)
+	DeleteSchool(http.ResponseWriter, *http.Request)
+	GetSchoolMembers(http.ResponseWriter, *http.Request)
+	PostSchoolMember(http.ResponseWriter, *http.Request)
+	DeleteSchoolMember(http.ResponseWriter, *http.Request)
```

Comment:
should we use `*EducationSchool`, similar to the `EducationUsers` and `EducationClass`?